pages/docs/some-react-code/index.jsx

```diff
@@ -3,18 +3,18 @@ import Demo from './_Demo'
 import DocumentTitle from 'react-document-title'
 import { config } from 'config'
 
+exports.data = {
+  title: 'Some React Code',
+  date: `${new Date()}`
```

Comment:
Hmm, I assume you mean the date? That would make sense. I copied that from the OP, and only tried the title. :)

Should touch base with the OP why he needed that perhaps.